Plugins/Json/src/AmbiguityConfigJsonConverter.cpp

```diff
@@ -0,0 +1,76 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023-2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Json/AmbiguityConfigJsonConverter.hpp"
+
+#include "Acts/AmbiguityResolution/ScoreBasedAmbiguityResolution.hpp"
+#include "Acts/Plugins/Json/ActsJson.hpp"
+
+#include <fstream>
+
+namespace Acts {
+
+std::pair<std::map<std::size_t, std::size_t>,
+          std::vector<ScoreBasedAmbiguityResolution::DetectorConfig>>
+from_json(const std::string& configFile) {
+  std::ifstream file(configFile);
+  if (!file.is_open()) {
+    std::cerr << "Error opening file: " << configFile << std::endl;
+    return {};
+  }
+
+  std::cout << "Reading configuration file: " << configFile << std::endl;
+
+  nlohmann::json j;
+
+  file >> j;
```

Comment:
Usually this function would take a `json` object and then turns it into something else. you can find an example here https://github.com/acts-project/acts/blob/bed53b2fd3ddffc36cc7c8176ea4bbb447e6ec2e/Plugins/Json/include/Acts/Plugins/Json/MaterialJsonConverter.hpp#L30

Reply:
Okay I will do it like that.

Reply:
I have changed the json to this style